README.md

```diff
@@ -234,5 +303,12 @@ Assuming that the server is running in cohttp's source directory:
 ```
 $ cohttp-curl-lwt 'http://0.0.0.0:8080/_oasis'
```

Comment:
May be update to `$ cohttp-curl-lwt 'http://0.0.0.0:8080/README.md`.

Reply:
well spotted